### PR TITLE
Add Transparent Dark theme

### PR DESCRIPTION
**Pull request by wgyt**
not sure if it works, adds a transparent theme

### CODE REVIEW

File: styles/themes/transparent.css [hunk 1]
```diff
@@ -0,0 +1,817 @@
...
+Copyright (c) 2018 GitHub Inc.
+https://github.com/primer/primitives/blob/main/LICENSE
+
+*/
+
+[data-theme='dark'] {
+  color-scheme: dark;
+}
+
+[data-theme='dark'] {
```
Comment (laymonage) [inline styles/themes/transparent.css:15]:
```suggestion
[data-theme='transparent'] {
  color-scheme: dark;
}

[data-theme='transparent'] {
```

**Comment by wgyt** [comment]
Currently testing this on my website, hopefully it works.

File: styles/themes/transparent.css [hunk 2]
```diff
@@ -0,0 +1,817 @@
+@import 'github-syntax-dark/lib/github-dark.css';
+
+/*
+
+MIT License
+Copyright (c) 2018 GitHub Inc.
+https://github.com/primer/primitives/blob/main/LICENSE
```
Comment (laymonage) [inline styles/themes/transparent.css:7]:
```suggestion
Modified from GitHub's dark theme in primer/primitives.

MIT License
Copyright (c) 2018 GitHub Inc.
https://github.com/primer/primitives/blob/main/LICENSE
```

**Review by laymonage — CHANGES_REQUESTED** [review]
Thanks! You also need to change the `[data-theme='dark']` selector for this to work.

Also, could you please change the theme name from `transparent` to `transparent_dark` (Transparent Dark)? We might add a "Transparent Light" theme in the future.

**Comment by wgyt** [comment]
> Thanks! You also need to change the `[data-theme='dark']` selector for this to work.
> 
> Also, could you please change the theme name from `transparent` to `transparent_dark` (Transparent Dark)? We might add a "Transparent Light" theme in the future.

Sure, I can change this.

**Review by laymonage — COMMENTED** [review]
Thanks, I probably should've made this clearer on the initial review :sweat_smile:

**Comment by wgyt** [comment]
It looks like it works :tada:
image

**Comment by laymonage** [comment]
@wgyt That's good to know, thanks!

One last thing, would you rather have the primary buttons (i.e. "Sign in with GitHub", "Comment", "Reply") have a background color, or just keep it transparent as it is now?

**Comment by wgyt** [comment]
> @wgyt That's good to know, thanks!
> 
> One last thing, would you rather have the primary buttons (i.e. "Sign in with GitHub", "Comment", "Reply") have a background color, or just keep it transparent as it is now?

I think that keeping them transparent is a good idea because this theme is supposed to be transparent

**Review by laymonage — APPROVED** [review]

**Comment by laymonage** [comment]
Thanks! :tada:

**Comment by nschonni** [comment]
Might want to check the contrast with something like https://wave.webaim.org/

**Comment by wgyt** [comment]
> Might want to check the contrast with something like https://wave.webaim.org/

Yeah, that's something I'll need to check on the website I used to test

**Comment by laymonage** [comment]
> Might want to check the contrast with something like https://wave.webaim.org/

@nschonni Yeah, but it really depends on the website that uses the theme. For example, a website with a light background will have a really bad contrast with this theme, and vice versa.

Even giscus' main website looks a bit weird with this theme as it uses `color-bg-canvas` (which is `transparent` in this theme). I think I'll have to make sure that the main page uses its own background color.

**Pull request merged**